pkg/unpackerr/handlers.go

```diff
@@ -222,7 +222,7 @@ func (u *Unpackerr) getDownloadPath(outputPath string, app starr.App, title stri
 	}
 
 	// Print the errors for each user-provided path.
-	u.Debugf("%s: Errors encountered looking for %s path: %q", app, title, errs)
```

Comment:
This is not a change we need here. A properly configured app won't produce this message. The final checked path's error is printed as an ERROR.